### DISCUSSION TITLE
Fatal INPUT FILE error, no valid exon lines in the GTF file

### DISCUSSION THREAD

**Discussion by bkffadia** [alexdobin/STAR#1718] [Q&A]
I run STAR using this command line
`!STAR --runMode genomeGenerate --genomeFastaFiles /content/drive/MyDrive/ncbi-genomes-2022-11-28/GRCh38_Cerrected.fna --sjdbGTFfile /content/drive/MyDrive/gencode.v34.annotation.gtf`

 and this message shows up :
**Fatal INPUT FILE error, no valid exon lines in the GTF file: /content/drive/MyDrive/gencode.v34.annotation.gtf
Solution: check the formatting of the GTF file. One likely cause is the difference in chromosome naming between GTF and FASTA file.**

I tried to change chromosome names in the FASTA file format so they become chr instead of chromosome and still get this message, am I doing somethig wrong ?

description in fasta file : NC_000001.11 Homo sapiens **chr1** GRCh38 Primary Assembly 248956422
GTF files : **chr1**\tHAVANA\ttranscript\t2560554\t2563504\t.\...

**Comment by alexdobin** [comment]
The FASTA file need to have the same chromosome names as the GTF, i.e. chr1, chr2 etc.
